src/app/games/trexo/TrexoTutorial.ts

```diff
@@ -0,0 +1,71 @@
+];
+let previousState: TrexoState = initialState;
+for (const move of moves) {
+    const state: TrexoState = TrexoRules.get().applyLegalMove(move, previousState);
+    previousState = state;
+}
+const firstMove: TrexoMove = TrexoMove.from(new Coord(5, 5), new Coord(5, 4)).get();
+const stateAtTurnOne: TrexoState = TrexoRules.get().applyLegalMove(firstMove, initialState);
+const secondMove: TrexoMove = TrexoMove.from(new Coord(5, 7), new Coord(5, 6)).get();
+const stateAtTurnTwo: TrexoState = TrexoRules.get().applyLegalMove(secondMove, stateAtTurnOne);
+
+export class TrexoTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        // 1. but du jeu et plateau
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`At Trexo, the goal of the game is to align 5 pieces of your color in a vertical, horizontal or diagonal line. However, the players play with tiles that are constituted of two pieces, one from each player!`,
+            TrexoState.getInitialState(),
+        ),
+        TutorialStep.anyMove(
+            $localize`Dropping a tile`,
+            $localize`When you drop at tile, it needs to be on even ground, and it cannot be right on top of another tile. In others words, it needs to be either on the floor, or on two tiles at the same height. To do that, just click on the square where you want to put the opponent side of the tile, then on the neighboring square where you want to put your piece.<br/><br/>You're playing Dark, put a tile on the board.`,
+            TrexoState.getInitialState(),
+            TrexoMove.from(new Coord(4, 4), new Coord(3, 4)).get(),
+            $localize`Congratulations!`,
+        ),
+        TutorialStep.fromMove(
+            $localize`Dropping a piece over other pieces`,
+            $localize`You can put a tile on other tiles. For that you must respect two rules:<br/><ul><li>The two pieces must be at the same level.</li><li>You cannot drop your piece on only one other piece.</li></ul><br/>You're playing Dark, do such a move!`,
+            stateAtTurnTwo,
+            [
+                TrexoMove.from(new Coord(5, 5), new Coord(5, 6)).get(),
+                TrexoMove.from(new Coord(5, 6), new Coord(5, 5)).get(),
+            ],
+            $localize`Congratulations!`,
+            $localize`Incorrect, you put that piece on the floor level, please put it on top of two pieces.`,
+        )
+            .withPreviousMove(secondMove)
+            .withPreviousState(stateAtTurnOne),
+        TutorialStep.fromPredicate(
+            $localize`Victory`,
+            $localize`To win at Trexo, you need to align 5 of your pieces, only the piece on top counts. Since you have to put the piece of your opponent first, if you create a victory for them and for you in the same turn, they'll be considered the winners (if you create only a victory for them too, obviously).<br/><br/>You're playing Dark, win.`,
```

Comment:
in the same turn -> at/during the same turn peut-être ?
C'est pas *dans* un tour mais à/durant un tour ? (en tout cas c'est pas en d'dans)